stubs/AMQPEnvelope.php

```diff
@@ -30,7 +30,7 @@ public function getRoutingKey()
     /**
      * Get the consumer tag of the message.
      *
-     * @return string The consumer tag of the message.
+     * @return string|null The consumer tag of the message.
```

Comment:
you could switch to real type declarations, can't you?

Reply:
Yup, that's true